test/evergreen.yml

```diff
@@ -4316,6 +4316,10 @@ buildvariants:
     - name: compile
     - name: make-check-test
```

Comment:
The make-check-test is actually not covered by the main PPC build variant. I think stress tests are the focus of these exotic platforms (unit-tests for sanity purpose).

Reply:
Good catch, that leaked through from the original CMake variant. Fixed in: cfac8c80bf5e520c50cb3b2871ac69cae7987dc6